packages/grid/src/vaadin-grid.js

```diff
@@ -912,13 +927,13 @@ class Grid extends ElementMixin(
 
     while (this.$.header.children.length < columnTree.length) {
       const headerRow = document.createElement('tr');
-      headerRow.setAttribute('part', 'row');
+      headerRow.setAttribute('part', 'row header');
```

Comment:
Ah, we intentionally left out the other specifiers but `row` initially to make it less likely that people try to style the `<tr>` elements (styles should primarily be applied to the cells).

Reply:
Good point, thanks. 
I think we can remove `header-row` and `footer-row` parts for now, as the issue does not mention them.

Reply:
Removed.